frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/PageAccordion.module.css

```diff
@@ -1,23 +1,23 @@
 .navigationMenu {
   display: flex;
-  justify-content: flex-start;
-  align-items: stretch;
+  align-items: center;
   padding-block: 12px;
-  margin-left: 10px;
+  padding: 3px;
```

Comment:
Can we use some of designsystemets spacing tokens here? 😄

Reply:
Changed it and removed padding-block, couldn't see the effect of it anymore.